internal/service-manager/client.go

```diff
@@ -19,10 +19,12 @@ import (
 )
 
 const (
-	componentName        = "ServiceManagerClient"
-	defaultSecret        = "sap-btp-service-operator"
-	defaultNamespace     = "kyma-system"
-	ServiceOfferingsPath = "/v1/service_offerings"
+	componentName                             = "ServiceManagerClient"
+	defaultSecret                             = "sap-btp-service-operator"
+	defaultNamespace                          = "kyma-system"
+	ServiceOfferingsPath                      = "/v1/service_offerings"
+	ServicePlansPath                          = "/v1/service_plans"
+	servicePlansForServiceOfferingQueryFormat = `?fieldQuery=service_offering_id+eq+%%27%s%%27`
```

Comment:
Cryptic, the reader is forced to evaluate this expression.
Comment could be useful, or usage of some function to "escape" parameters in ordinary string.

Reply:
I agree, query format const may not be understandable without the context:
https://help.sap.com/docs/service-manager/sap-service-manager/filtering-parameters-and-operators
I will try to replace it with something cleaner.

Reply:
Suggestion applied.